app/src/main/java/org/fossasia/pslab/fragment/LuxMeterFragmentConfig.java

```diff
@@ -0,0 +1,161 @@
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.AdapterView;
+import android.widget.EditText;
+import android.widget.SeekBar;
+import android.widget.Spinner;
+
+import org.fossasia.pslab.R;
+import org.fossasia.pslab.communication.ScienceLab;
+import org.fossasia.pslab.communication.peripherals.I2C;
+import org.fossasia.pslab.communication.sensors.BH1750;
+import org.fossasia.pslab.others.ScienceLabCommon;
+
+import java.io.IOException;
+
+public class LuxMeterFragmentConfig extends Fragment {
+
+    EditText highLimit;
+    EditText updatePeriod;
+    SeekBar highLimitSeek;
+    SeekBar updatePeriodSeek;
+    Spinner gainValue;
+    Spinner selectSensor;
+    CardView gainRangeCard;
+    private static ScienceLab scienceLab;
+    private static BH1750 bh1750;
+
+    public static ScienceLab getScienceLab() {
+        return scienceLab;
+    }
+
+    final int highLimitMax = 1000;
+    final int updatePeriodMax = 980;
+    private static int highValue = 0;
+    private static int updatePeriodValue = 20;
+    public LuxMeterFragmentConfig() {
+        // Required empty public constructor
+    }
+
+    public static LuxMeterFragmentConfig newInstance() {
+        return new LuxMeterFragmentConfig();
+    }
+
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        View view = inflater.inflate(R.layout.fragment_lux_meter_config, container, false);
+
+        highLimit = (EditText) view.findViewById(R.id.lux_hight_limit_text);
+        highLimitSeek = (SeekBar) view.findViewById(R.id.lux_hight_limit_seekbar);
+        updatePeriod = (EditText) view.findViewById(R.id.lux_update_period_text);
+        updatePeriodSeek = (SeekBar) view.findViewById(R.id.lux_update_period_seekbar);
+        gainValue = (Spinner) view.findViewById(R.id.spinner_bh1750_gain);
+        selectSensor = (Spinner) view.findViewById(R.id.spinner_select_light);
+        gainRangeCard = (CardView) view.findViewById(R.id.cardview_gain_range);
+
+        highLimitSeek.setMax(highLimitMax);
+        updatePeriodSeek.setMax(updatePeriodMax);
+
+        highLimit.setText(String.format("%d", highValue));
+        highLimitSeek.setProgress(highValue);
+        updatePeriod.setText(String.format("%d" ,updatePeriodValue+20));
+        updatePeriodSeek.setProgress(updatePeriodValue+20);
+
+        highLimitSeek.setOnSeekBarChangeListener(new SeekBar.OnSeekBarChangeListener() {
+            @Override
+            public void onProgressChanged(SeekBar seekBar, int progress, boolean fromUser) {
+                highValue = progress;
+                highLimit.setText(String.format("%d", progress));
+            }
+
+            @Override
+            public void onStartTrackingTouch(SeekBar seekBar) {
+                //do nothing
```

Comment:
These type of comments are not needed. Just leave it empty

Reply:
@Avjeet What do you say? For codacy it does need

Reply:
Well the comments makes it easy for review and also for understanding the code in future.